test/verify/check-machines

```diff
@@ -30,37 +30,111 @@ def readFile(name):
             content = f.read().replace('\n', '')
     return content
 
+SPICE_XML="""
+    <video>
+      <model type='vga' heads='1' primary='yes'/>
+      <alias name='video0'/>
+    </video>
+    <graphics type='spice' port='5900' autoport='yes' listen='127.0.0.1'>
+      <listen type='address' address='127.0.0.1'/>
+      <image compression='off'/>
+    </graphics>
+"""
+
+VNC_XML="""
+    <graphics type='vnc' port='5900' autoport='yes' listen='127.0.0.1'>
+      <listen type='address' address='127.0.0.1'/>
+    </graphics>
+    <video>
+      <model type='vga' heads='1' primary='yes'/>
+      <alias name='video0'/>
+    </video>
+"""
+
+DOMAIN_XML="""
+<domain type='kvm'>
+  <name>{name}</name>
+  <vcpu>1</vcpu>
+  <os>
+    <type arch='x86_64'>hvm</type>
+    <boot dev='hd'/>
+    <boot dev='network'/>
+  </os>
+  <memory unit='MiB'>768</memory>
+  <currentMemory unit='MiB'>768</currentMemory>
+  <features>
+    <acpi/>
+  </features>
+  <devices>
+    <disk type='block' device='disk'>
+      <driver name='qemu' type='raw'/>
+      <source dev='/dev/{disk}'/>
+      <target dev='hda' bus='ide'/>
+      <serial>ROOT</serial>
+    </disk>
+    <disk type='file' snapshot='external'>
+      <driver name='qemu' type='qcow2'/>
+      <source file='{image}'/>
+      <target dev='hdb' bus='ide'/>
+      <serial>SECOND</serial>
+    </disk>
+    <controller type='scsi' model='virtio-scsi' index='0' id='hot'/>
+    <interface type='network'>
+      <source network='default' bridge='virbr0'/>
+      <target dev='vnet0'/>
+      <model type='rtl8139'/>
+    </interface>
+    <console type='pty'>
+      <target type='serial' port='0'/>
+      <log file='/var/log/libvirt/qemu/console-{name}.log' append='on'/>
```

Comment:
This might work:
```
    <console type='file'>
      <target type='serial' port='0'/>
      <source path='/var/log/libvirt/qemu/console-{name}.log' append='on'/>
    </console>
```
It works on fedora-26, not tested anywhere else.